README.md

```diff
@@ -819,6 +819,17 @@ As a consequence, the work might end up being very fragmented and each chunk may
 
 Some projects do not mind this however, so this solution may still be quite suitable.
 
+## Jobs admin UI
```

Comment:
```suggestion
## Enabling advanced admin UI
```

Reply:
How about 'Enabling extended admin UI' instead of 'Advanced'? Even though this is the readme, we try to avoid advanced wording coupled in UI contexts where possible.